src/page-modules/contact/means-of-transport/means-of-transport-form-machine.ts

```diff
@@ -77,6 +79,12 @@ export const meansOfTransportFormMachine = setup({
         } else {
           context.errorMessages[inputName] = [];
         }
+
+        if (inputName === 'transportMode')
+          return setTransportModeAndResetLineAndStops(context, value);
+
+        if (inputName === 'line') return setLineAndResetStops(context, value);
+
```

Comment:
I think that we should create separate functions when we have special cases like this. That is more explicit in my opinion and also easier to maintain.

Reply:
Moved both `setTransportModeAndResetLineAndStops ` and `setLineAndResetStops ` to their own actions✅